doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -192,7 +192,8 @@ nRF5340 Audio
 nRF Machine Learning (Edge Impulse)
 -----------------------------------
 
-|no_changes_yet_note|
+* Updated Bluetooth advertising data and scan response data logic.
+  The Bluetooth device name is now included in the scan response data only if used Bluetooth local identity has no bond.
```

Comment:
Shouldn't we add the use of "config BT_LE_ADV_PROV_DEVICE_NAME" Kconfig option also. I think that is now used to scan the response data.

Reply:
I think that this is more like an implementation detail. The generic information that we stated to use Bluetooth LE advertising data providers to configure the advertising data and scan response data would be included here: https://github.com/nrfconnect/sdk-nrf/pull/8457.
Can we leave it as it is?

Reply:
may be:
```suggestion
  The Bluetooth device name is now included in the scan response data, using the :ref:`CONFIG_BT_LE_ADV_PROV_FLAGS <config_desktop_app_options>` option, only if the used Bluetooth local identity has no bond.
```
Same applies for the other entries as well.
I am fine if the information about the scan response data is included as part of PR: https://github.com/nrfconnect/sdk-nrf/pull/8457.

Reply:
The `CONFIG_BT_LE_ADV_PROV_FLAGS` is an NCS option, so we should use :kconfig:option: syntax.

I am not sure if we need to go into details and mention Kconfig option names in the release notes (I think we could just inform in nRF Desktop documentation where the advertising data configuration is located). User would need to get familiar with new configuration method anyway. I try to keep the release notes simple and avoid going into details here.
If you insist on adding direct Kconfig references here, I could add them of course

Reply:
I also had an offline conversation with @greg-fer, and we could keep the content as is if we add the information to the nRF Desktop and other pages, whether this is relevant. 